package.json

```diff
@@ -30,41 +34,43 @@
   },
   "homepage": "https://github.com/Storytel/react-redux-spinner#readme",
   "dependencies": {
-    "@types/nprogress": "~0.0.29",
-    "prop-types": "^15.5.7"
+    "@types/nprogress": "^0.0.29",
+    "nprogress": "^0.2.0",
+    "prop-types": "^15.6.2"
   },
   "peerDependencies": {
     "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
     "react-dom": "^0.14 || ^15.0.0 || ^16.0.0",
+    "react-redux": "*",
     "redux": "^3.0.0 || ^4.0.0"
   },
   "devDependencies": {
-    "@types/react": "~16.0.25",
-    "babel-cli": "^6.6.5",
-    "babel-core": "^6.7.2",
-    "babel-eslint": "^8.0.2",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "css-loader": "^0.28.7",
-    "eslint": "^4.11.0",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.2.3",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.0.0",
+    "@types/react": "^16.7.18",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.4",
+    "css-loader": "^2.1.0",
+    "eslint": "^5.11.1",
     "eslint-config-storytel": "^4.0.0",
     "eslint-plugin-react": "^7.5.1",
     "expect.js": "^0.3.1",
-    "extract-text-webpack-plugin": "^3.0.2",
-    "file-loader": "^1.1.5",
-    "html-webpack-plugin": "^2.12.0",
-    "mocha": "^4.0.1",
-    "nprogress": "^0.2.0",
+    "file-loader": "^3.0.1",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.5.0",
+    "mocha": "^5.2.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.1",
```

Comment:
Why are those two new dependency required?
- `optimize-css-assets-webpack-plugin`
- `html-webpack-plugin`

Reply:
HtmlWebpackPlugin is used to generate an index.html page
MiniCssExtractPlugin is used to extract CSS from the webpack bundle
OptimizeCssAssetsWebpackPlugin minifies and dedupes the extracted CSS

Reply:
Ok. But those are new additions right? I was wondering if this was related to the changes towars the new context api, or if it was an unrelated improvement.